tests/typecheck/fields/test_related.yml

```diff
@@ -1088,6 +1089,7 @@
 
         reveal_type(MyModel.auto_through.through)
         reveal_type(MyModel.auto_through.through.mymodel)
+        reveal_type(MyModel.auto_through.through._default_manager)
```

Comment:
What about other m2m fields?

Reply:
You mean in addition to `_default_manager`? Or which fields?

Reply:
Ping @sobolevn? This solves the issue I had.